dbt/adapters/databricks/connections.py

```diff
@@ -728,30 +735,37 @@ class DatabricksDBTConnection(Connection):
     compute_name: str = ""
     http_path: str = ""
     thread_identifier: Tuple[int, int] = (0, 0)
+    max_idle_time: int = DEFAULT_MAX_IDLE_TIME
 
     def _acquire(self, node: Optional[ResultNode]) -> None:
         """Indicate that this connection is in use."""
         logger.debug(f"DatabricksDBTConnection._acquire: {self._get_conn_info_str()}")
         self._log_usage(node)
-        # Make sure this won't be cleaned up as being idle for too long.
-        self.last_used_time = None
         self.acquire_release_count += 1
 
     def _release(self) -> None:
         """Indicate that this connection is not in use."""
         logger.debug(f"DatabricksDBTConnection._release: {self._get_conn_info_str()}")
-        self.last_used_time = time.time()
         # Need to check for > 0 because in some situations the dbt code will make an extra
         # release call on a connection.
         if self.acquire_release_count > 0:
             self.acquire_release_count -= 1
 
+        if self.acquire_release_count == 0:
```

Comment:
If, as stated above, sometimes dbt makes extra release calls, is it safe to have this based on release_count == 0?  Is this counter ever above 1?  If so, doesn't that mean we could hit 0 when we should actually be at 1?